packages/typescript-config/tsconfig.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "display": "React Native",
+    "compilerOptions": {
+      "target": "esnext",
+      "module": "commonjs",
+      "types": ["react-native", "jest"],
+      "lib": [
+        "es2019",
+        "es2020.bigint",
+        "es2020.date",
+        "es2020.number",
+        "es2020.promise",
+        "es2020.string",
+        "es2020.symbol.wellknown",
+        "es2021.promise",
+        "es2021.string",
+        "es2021.weakref",
+        "es2022.array",
+        "es2022.object",
+        "es2022.string"
+      ],
+      "allowJs": true,
+      "jsx": "react-native",
+      "noEmit": true,
+      "isolatedModules": true,
+      "strict": true,
+      "moduleResolution": "nodenext",
+      "resolveJsonModule": true,
+      "allowSyntheticDefaultImports": true,
+      "esModuleInterop": true,
+      "skipLibCheck": true,
+      // Causes issues with package.json "exports"
```

Comment:
@NickGerleman apologies for the ping - do you know what the specifics of the issue mentioned here are?

It looks like you originally changed it in the old repo but the commit message doesn't explain why: https://github.com/tsconfig/bases/commit/5b4e858c6ed774a2e1144d9640c93c24f75a2f4a